src/backoffice/views.py

```diff
@@ -1,18 +1,16 @@
 import logging
 from itertools import chain
+from datetime import datetime
```

Comment:
Probably doesn't matter since we are only getting the year, but I would use `from django.utils import timezone` instead.